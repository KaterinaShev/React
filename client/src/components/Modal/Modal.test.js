import { render } from '@testing-library/react'
import Modal from './Modal'

const close = jest.fn()

describe('Modal tests', () => {
  test('Modai is render', () => {
    const { asFragment } = render(<Modal onClose={close} text='Text modal' />)
    expect(asFragment()).toMatchSnapshot()
  })
})