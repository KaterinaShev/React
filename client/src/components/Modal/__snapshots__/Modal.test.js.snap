// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal tests Modai is render 1`] = `
<DocumentFragment>
  <div
    class="bg"
  />
  <div
    class="modal"
  >
    <div
      class="modal_header"
    >
      <div
        class="modal_close"
      >
        <svg>
          close.svg
        </svg>
      </div>
    </div>
    <div
      class="modal_body"
    >
      <p>
        Text modal
      </p>
      <div
        class="modal_footer"
      />
    </div>
  </div>
</DocumentFragment>
`;
